common/templates/discussion/index.html

```diff
@@ -28,4 +30,25 @@
 
 <%include file="_discussion_course_navigation.html" args="active_page='discussion'" />
 
-<%include file="/discussion/_discussion_course.html" />
+<section class="discussion container" id="discussion-container"
+         data-roles="${roles}"
+         data-course-id="${course_id | h}"
+         data-user-info="${user_info}"
+         data-threads="${threads}"
+         data-thread-pages="${thread_pages}"
+         data-content-info="${annotated_content_info}"
+         data-sort-preference="${sort_preference}"
+         data-flag-moderator="${flag_moderator}"
+         data-user-cohort-id="${user_cohort}"
+         data-course-settings="${course_settings}">
+    <div class="discussion-body">
+        <div class="forum-nav"></div>
+        <div class="discussion-column">
+          <article class="new-post-article" style="display: none"></article>
+          <div class="forum-content"></div>
+        </div>
+    </div>
+</section>
```

Comment:
@e-kolpakov Could this section be moved to a separate subtemplate and `<%include%>`-d here and in https://github.com/open-craft/edx-platform/blob/37bee2edde4d159310c560579db5efb8c3eb0c3a/common/templates/discussion/_discussion_course.html#L24-L42? It looks the same except that the `data-course-id` value is passed through the `| h` filter here, but not in `_discoussion_course.html` - is that intentional?